docs/Env_Setup.rst

```diff
@@ -108,6 +108,188 @@ with a minimal fedora 23 installation. It will leave any logs under the `logs`
 directory and any generated artifacts under `exported-artifacts`.
 
 
-
    .. _`official yapf repo`: https://github.com/google/yapf
    .. _`copr`: https://copr.Fedoraproject.org/coprs/fulltext/?fulltext=yapf
+
+
+Running Lago in virtualenv
+---------------------------
+This was tested on a fresh Fedora-24 Cloud image, but should work on Fedora 23
+and most likely with little package-names resolving on CentOS >7.
+
+Install dependencies::
+
+  > sudo dnf install -y redhat-rpm-config libvirt libvirt-devel \
+     libguestfs-devel libguestfs-tools qemu-img qemu-kvm \
+     libvirt-daemon git gcc libffi-devel python-devel \
+     openssl-devel bats python-magic python-libguestfs
+
+Note that ``python-magic`` and ``python-libguestfs`` are currently not
```

Comment:
On Debian, lago works well with `file-magic` pip package. This package is based on the official Github [mirror](https://github.com/file/file). One author of this package is Reuben Thomas who is listed as a [contributor](https://www.openhub.net/p/file/contributors/summary) of file.
Currently, `file-magic` pip package 0.3.0 corresponds to file 5.28.